asreview/state/utils.py

```diff
@@ -122,11 +127,43 @@ def state_from_file(data_fp):
         logging.error(f"File {data_fp} does not exist, cannot create state.")
         return None
 
-    if not Path(data_fp).suffix in STATE_EXTENSIONS:
-        logging.error(f"file {data_fp} does not end with {STATE_EXTENSIONS}.")
-        return None
+    if Path(data_fp).suffix == ".asreview":
+        base_state = state_from_dot_asreview(data_fp)
+    else:
+        if not Path(data_fp).suffix in STATE_EXTENSIONS:
+            logging.error(f"file {data_fp} does not end with {STATE_EXTENSIONS}.")
+            return None
+        base_state = _get_state_class(data_fp)(state_fp=data_fp, read_only=True)
```

Comment:
Not tested, but what about this? And raise ValueError instead of logging?

```
    if Path(data_fp).suffix == ".asreview":
        base_state = state_from_dot_asreview(data_fp)
    else if Path(data_fp).suffix in STATE_EXTENSIONS:
        base_state = _get_state_class(data_fp)(state_fp=data_fp, read_only=True)
    else:
        raise ValueError(f"Expected ASReview file or file {data_fp} with extension {STATE_EXTENSIONS}.")
```